### ISSUE TITLE
bbc-a11y fails test for 'Element is invisible on focus' for scrollable nav on RTL languages

### ISSUE THREAD

**Issue by tochwill** [bbc/simorgh#5222]
**Describe the bug**
There is a test failure when running bbc-a11y for the persian site. This is because there is a test that every focusable element is on screen when it is focused. The browser behaviour (for chrome at least, which is running these tests) for horizontally scrollable items, is that the item is brought fully into view if it begins 50% or more off screen, but remains as it is (with some of the element off screen) if it is already 50% on screen. The bbc-a11y test passes for LTR languages because it checks that the left edge of the element is on screen on focus - this is the case for the LTR nav, but the left most edge remains off screen for some links on the RTL site. We have currently disabled this bbc-a11y test. We should either override the browser behaviour to ensure the full link is brought into view on focus if possible, or see if we can implement the bbc-a11y test in a different way.

**To Reproduce**

1. `npm run bbcA11y` in Simorgh, after removing the exception for `//div[@id='root']/header/nav/div/div[1]/div/ul` from the `a11y.js` config file.

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Additional context**

**Comment by DenisHdz** [comment]
We have ignored the test for now https://github.com/bbc/simorgh/pull/5223

**Comment by tochwill** [comment]
Blocked on https://github.com/bbc/simorgh/pull/4833

**Comment by greenc05** [comment]
@tochwill please could you create an issue in bbc-a11y and assign to @jroebu14  https://github.com/bbc/bbc-a11y/issues

**Comment by jroebu14** [comment]
@tochwill @greenc05 thought I'd make a head start before you created the issue in bbc-a11y. Not tested yet but [this may fix the issue ](https://github.com/bbc/bbc-a11y/pull/312/files?w=1)

These lines:
```js
var dir = $('html').attr('dir') === 'rtl' ? 'right' : 'left';
```
```js
offset[dir] + $element.width() > 0;
```

**Comment by jroebu14** [comment]
@tochwill I checked out and ran new navigation PR and tested Persian but the bbc-a11y passes for everything i.e. I can't replicate the error you're seeing. Can you let me know specifically what test fails for you? Thanks

**Comment by tochwill** [comment]
Hey! @jroebu14 yes that would fix the issue - thank you for taking a look! We've disabled that element for now in the navigation PR so the build would pass - we have an exception for `//div[@id='root']/header/nav/div/div[1]/div/ul` in the `a11y.js` file. I can still raise the issue today :)
